content/techniques/serialization.md

```diff
@@ -100,6 +100,45 @@ Options passed via `@SerializeOptions()` are passed as the second argument of th
 
 A working example is available [here](https://github.com/nestjs/nest/tree/master/sample/21-serializer).
 
+#### Sequelize
+
+class-transformer does not play well with Sequelize, so you're going to need a few extra steps to get it to work properly.
+
+First off, create a response DTO with @Expose / @Exclude
+```typescript
+import { Exclude, Expose } from 'class-transformer';
+
+@Exclude()
+export class UserResponseDto {
+    @Expose()
+    name: string;
+    
+    @Expose()
+    email: string;
+
+    password: string;
+}
+```
+
+Then use `plainToClass(DTO, Promise<Model>)` in your controller:
+```typescript
+    @Get(':id')
+    @UseInterceptors(ClassSerializerInterceptor)
+    findOne(@Param('id') id: string): UserResponseDto {
+        return plainToClass(
+            UserResponseDto,
+            this.usersService.findOneById(+id),
+        );
+    }
```

Comment:
Personally I would prefer we show this happening in the service rather than the controller, to stay with the idea of thin controllers fat services, and keeping business logic out of the controllers

Reply:
That's fair, however my reasoning for putting it in the controller is that calls to findOneById could be used by an auth service, in which case the password field would be required.